_docs/sql-reference/sql-commands/007-analyze-table-refresh-metadata.md

```diff
@@ -29,6 +30,10 @@ The ANALYZE TABLE REFRESH METADATA statement supports the following syntax:
 The name of the table or directory for which Drill will collect table metadata. If the table does not exist, the table
  is temporary or if you do not have permission to read the table, the command fails and metadata is not collected and stored.
 
+*table({table function name}(parameters))*
```

Comment:
Well, first of all this syntax is only available since Drill 1.18.
Secondly, we do need some example for the users.

Reply:
Thanks, added an example and specified the version where it is available.